### DISCUSSION TITLE
How to disable code warnings?

### DISCUSSION THREAD

**Discussion by ituri** [LazyVim/LazyVim#2069] [Q&A]
Hello,

As an absolute beginner in Neovim/Lazyvim, I've been been trying my best to answer all upcoming questions using the documentation. However, I'm having trouble finding an answer to this particular issue. I know it might be a simple question, so please forgive me.
In my markdown files, I'm receiving these warnings:

![CleanShot 2023-11-26 at 08 28 53](https://github.com/LazyVim/LazyVim/assets/871297/c604c1df-79ac-47fb-84be-bc66c0aa9847)

Can someone enlighten me as to which plugin is responsible for generating these warnings? I'd like to toggle them on/off, but I'm having trouble identifying the plugin. Any help would be much appreciated!

**Comment by dpetka2001** [comment]
I'm not sure, but I'm guessing these diagnostics come from either `marksman` or `markdownlint`. LazyVim has a keymap for toggling diagnostics on/off, just press `<leader>ud`.

**Reply by nmvega** [reply]
This doesn't work for me. It does say disabled when I perform the keystrokes (space + u + d), but the warnings remain. :(

**Comment by ituri** [comment]
Thank you, that's it!
